File: engine/engine-model/src/main/java/tools/packet/showitemgaininchat/ShowWheelsLeft.java
package tools.packet.showitemgaininchat;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowWheelsLeft(Integer remaining) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_ITEM_GAIN_IN_CHAT;
   }
}

File: engine/model/src/main/scala/tools/packet/showitemgaininchat/ShowWheelsLeft.scala
package tools.packet.showitemgaininchat

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowWheelsLeft(private var _remaining: Int) extends PacketInput {
  def remaining: Int = _remaining

  override def opcode(): SendOpcode = SendOpcode.SHOW_ITEM_GAIN_INCHAT
}